test/integration/policy-imagemanifest_test.go

```diff
@@ -0,0 +1,123 @@
+		By("Creating the policy on the Hub")
+		_, err := utils.KubectlWithOutput(
+			"apply", "-f", policyIMVURL, "-n", userNamespace, "--kubeconfig="+kubeconfigHub,
+		)
+		Expect(err).To(BeNil())
+
+		By("Patching the namespaceSelector to use the " + userNamespace + " namespace")
+		_, err = clientHubDynamic.Resource(common.GvrPolicy).Namespace(userNamespace).Patch(
+			context.TODO(),
+			policyIMVName,
+			k8stypes.JSONPatchType,
+			[]byte(`[{"op": "replace", "path": "/spec/policy-templates/0/objectDefinition/spec/namespaceSelector/include", "value": ["`+userNamespace+`"]}]`),
+			metav1.PatchOptions{},
+		)
+		Expect(err).To(BeNil())
+
+		By("Patching placement rule")
+		err = common.PatchPlacementRule(
+			userNamespace, "placement-"+policyIMVName, clusterNamespace, kubeconfigHub,
+		)
+		Expect(err).To(BeNil())
+
+		By("Checking that " + policyIMVName + " exists on the Hub cluster")
+		rootPolicy := utils.GetWithTimeout(
+			clientHubDynamic, common.GvrPolicy, policyIMVName, userNamespace, true, defaultTimeoutSeconds,
+		)
+		Expect(rootPolicy).NotTo(BeNil())
+	})
+
+	It("stable/"+policyIMVName+" should be created on managed cluster", func() {
+		By("Checking the policy on managed cluster in ns " + clusterNamespace)
+		managedPolicy := utils.GetWithTimeout(
+			clientManagedDynamic,
+			common.GvrPolicy,
+			userNamespace+"."+policyIMVName,
+			clusterNamespace,
+			true,
+			defaultTimeoutSeconds,
+		)
+		Expect(managedPolicy).NotTo(BeNil())
+	})
+
+	It("stable/"+policyIMVName+" should be NonCompliant", func() {
+		By("Checking if the status of the root policy is NonCompliant")
+		Eventually(
+			common.GetComplianceState(clientHubDynamic, userNamespace, policyIMVName, clusterNamespace),
+			defaultTimeoutSeconds*2,
+			1,
+		).Should(Equal(policiesv1.NonCompliant))
+	})
+
+	It("Enforcing stable/"+policyIMVName, func() {
+		By("Patching remediationAction = enforce on the root policy")
+		_, err := clientHubDynamic.Resource(common.GvrPolicy).Namespace(userNamespace).Patch(
+			context.TODO(),
+			policyIMVName,
+			k8stypes.JSONPatchType,
+			[]byte(`[{"op": "replace", "path": "/spec/remediationAction", "value": "enforce"}]`),
+			metav1.PatchOptions{},
+		)
+		Expect(err).To(BeNil())
+	})
+
+	It("The subscription should exist", func() {
+		By("Checking the subscription in the " + userNamespace + " namespace")
+		Eventually(
+			func() error {
+				_, err := common.OcHub("get", "subscriptions", "-n", userNamespace, subName, "-o", "name")
+
+				return err
+			},
+			defaultTimeoutSeconds*2,
+			1,
+		).Should(BeNil())
+	})
+
+	It("stable/"+policyIMVName+" should be NonCompliant", func() {
+		By("Checking if the status of the root policy is NonCompliant")
+		Eventually(
+			common.GetComplianceState(clientHubDynamic, userNamespace, policyIMVName, clusterNamespace),
+			defaultTimeoutSeconds*2,
+			1,
+		).Should(Equal(policiesv1.NonCompliant))
+	})
+
+	It("Cleans up", func() {
```

Comment:
Also need to remove the csv so that operator will be fully uninstalled as well as the CRDs